build.zig

```diff
@@ -647,6 +647,13 @@ fn addCompilerStep(b: *std.Build, options: AddCompilerStepOptions) *std.Build.St
         .strip = options.strip,
         .sanitize_thread = options.sanitize_thread,
         .single_threaded = options.single_threaded,
+        .code_model = switch (options.target.result.cpu.arch) {
+            // FIXME:
```

Comment:
This comment should be changed to explain the relocation range problem we're working around by changing the code model. I don't think it needs to be a FIXME.

Reply:
Fixed the comment based on your suggestion.